processing/src/test/java/org/apache/druid/java/util/common/StringUtilsTest.java

```diff
@@ -386,4 +386,19 @@ public void testJavaStringCompare()
       }
     }
   }
+
+  @Test(expected = NullPointerException.class)
```

Comment:
nit: maybe use `Assert.assertThrows` instead of expected exceptions

Reply:
fixed